runtime/src/accounts.rs

```diff
@@ -332,7 +338,7 @@ impl Accounts {
         );
 
         let mut versions: Vec<(Pubkey, u64, B)> = accumulator.into_iter().flat_map(|x| x).collect();
-        versions.sort_by_key(|s| (s.0, s.1));
+        versions.par_sort_by_key(|s| (s.0, s.1));
```

Comment:
do we want to install() this code in this thread's rayon context?

Reply:
not sure. would have to test it. I'm not sure when it helps @pgarg66

Reply:
I think the last round of experimentation with rayon indicated to us that we *never* want to use the default pool, because lots of other modules may also be doing so...  we want a per-thread pool, at a minimum

Reply:
ok. i'll put a separate thread pool here.

Reply:
replay_stage calls this, already has a pool for entries, could be re-used if our rayon pools came from a solana rayon wrapper that put pools in TLS...